errored.go

```diff
@@ -152,11 +152,13 @@ func (e *Error) Error() string {
 	}
 
 	if e.trace || AlwaysTrace {
-		ret := desc + "\n"
-
-		for _, stack := range e.stack {
-			for _, line := range stack {
-				ret += fmt.Sprintf("\t%s [%s %d]\n", line.fun, line.file, line.line)
+		var ret string
+		for i := 0; i < len(e.errors); i++ {
+			ret += fmt.Sprintf("%s\n", e.errors[i].Error())
+			if len(e.stack) > i {
```

Comment:
It should be doing this and I'm not sure why it isn't. I'll look into it.

I'll paste this test into my branch unless you want the commit; then you can PR me. :D